pkg/ebpf/c/vmlinux.h

```diff
@@ -669,13 +670,64 @@ struct dentry {
     struct inode *d_inode;
 };
 
+enum bpf_func_id
```

Comment:
could you also move BPF_FUNC_probe_write_user and BPF_FUNC_override_return here from missing_definitions.h?